Application/EdFi.Ods.AdminApi.AdminConsole/Infrastructure/Services/Tenants/TenantService.cs

```diff
@@ -67,7 +65,7 @@ public async Task<List<TenantModel>> GetTenantsAsync(bool fromCache = false)
                 var connectionString = tenantConfig.Value.ConnectionStrings.First(p => p.Key == ADMIN_DB_KEY).Value;
                 if (!ConnectionStringHelper.ValidateConnectionString(_appSettings.AppSettings.DatabaseEngine!, connectionString))
                 {
-                    _log.Warn($"Tenant {tenantConfig.Key} has a wrong connection string for database {ADMIN_DB_KEY}");
+                    _log.WarnFormat("Tenant {Key} has a wrong connection string for database {ADMIN_DB_KEY}", tenantConfig.Key, ADMIN_DB_KEY);

```

Comment:
```suggestion
                    _log.WarnFormat("Tenant {Key} has an invalid connection string for database {ADMIN_DB_KEY}", tenantConfig.Key, ADMIN_DB_KEY);
```

Also, is it possible to inject the database engine name? That might be a strong clue as to the problem, i.e. someone using an MSSQL style connection string with PostgreSQL. You might be able to get the database engine name from `_appSettings`.

Reply:
The engine has been added:

```
_log.WarnFormat("Tenant {Key} has an invalid connection string for database {ADMIN_DB_KEY}. Database engine is {engine}",   tenantConfig.Key, ADMIN_DB_KEY, _appSettings.AppSettings.DatabaseEngine);
```